adafruit_sht31d.py

```diff
@@ -256,7 +263,7 @@ def _read(self):
         return self._cached_temperature, self._cached_humidity
 
     @property
-    def mode(self):
+    def mode(self) -> str:
```

Comment:
Since only certain `str` arguments are allowed, it's preferable to add that information into the type annotation.  Let's type annotate this return (and the argument of `value` in the setter below) as `Literal["Single", "Periodic"]`.

Reply:
Since `Literal` isn't available in `typing` for Python 3.7, import it from `typing_extensions` within the `typing` import block.